service/mapreduce_service.cc

```diff
@@ -443,6 +443,7 @@ future<query::mapreduce_result> mapreduce_service::execute_on_this_shard(
     auto rs_builder = cql3::selection::result_set_builder(
         *selection,
         now,
+        nullptr,
```

Comment:
I added a new parameter for the `query_options` needed to evaluate bind variables in selectors, but it's not needed in this context and the options are empty anyway. and I can't ignore it and use the default value because there's another parameter after that.
But actually I see now that the parameter after that also has the default value, so I could remove both of them.